openmetadata-docs/content/v1.5.x/menu.md

```diff
@@ -2089,4 +2089,12 @@ site_menu:
     url: /sdk/java
   - category: SDK / Go
     url: /sdk/go
+
+  - category: Metapilot
+    url: /metapilot
+    isCollateOnly: true
+  
+   category: Metapilot / How to Use MetaPilot
+    url: /metapilot/how-to-use-metapilot
+    isCollateOnly: true
```

Comment:
Let's remove it from `menu.md` and add it in `collate-menu.md`

Reply:
ok on it